var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -42,7 +42,7 @@ class Dealii(CMakePackage):
     version('8.3.0', 'fc6cdcb16309ef4bea338a4f014de6fa')
     version('8.2.1', '71c728dbec14f371297cd405776ccf08')
     version('8.1.0', 'aa8fadc2ce5eb674f44f997461bf668d')
-    version('develop', git='https://github.com/dealii/dealii.git')
```

Comment:
why did you need this change? It should (and it does as far as I know) checkout `master` branch by default.